src/index.ts

```diff
@@ -30,69 +30,48 @@ const extractTokenFromHeader = (header: string | null): string | null => {
   return token ?? null;
 };
 
-class AuthProvider {
-  constructor(private url: string) {}
-
-  async checkToken(token: string) {
-    const query = gql`
-      query checkToken($token: String!) {
-        checkToken(token: $token) {
-          isValid
-        }
-      }
-    `;
-
-    const result = await request<{ checkToken: { isValid: boolean } }>(
-      this.url,
-      query,
-      {
-        token,
-      }
-    );
-
-    return result.checkToken.isValid;
-  }
-}
-
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const authProvider = new AuthProvider(process.env.USER_OFFICE_BACKEND!);
```

Comment:
Maybe you could use non-null assertion operator (!) in both places to keep it consistent and remove the eslint warning suppressor if possible
const authProvider = new AuthProvider(process.env.USER_OFFICE_BACKEND!)!;